bambi/models.py

```diff
@@ -108,49 +120,41 @@ def build(self, backend=None):
                 and look in self._backend_name.
         '''
 
+        # retain only the complete cases
+        n_total = len(self.data.index)
+        if len(self.completes):
+            completes = [set(x) for x in sum(self.completes, [])]
+            completes = set.intersection(*completes)
+        else:
```

Comment:
I'm confused by the `len(self.completes)` check; suppose there's a case where NA-checking is done, but 100% of rows have at least one missing value. Then won't `self.completes` also be empty? Do we want then want to use all data anyway in that situation?

If the above is correct (and I'm probably missing something), we could probably default to `self.completes = None` and check that `is not None` rather than empty.